### DISCUSSION TITLE
Select install mode.

### DISCUSSION THREAD

**Discussion by MikeyBronowski** [JanDeDobbeleer/oh-my-posh#2199] [Troubleshoot]
When running this 

`Set-ExecutionPolicy Bypass -Scope CurrentUser -Force; Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://ohmyposh.dev/install.ps1'))`

I get a prompt, which possibly breaks my AutomatedLab setup, as I cannot click there 

![image](https://user-images.githubusercontent.com/13431760/166125923-f236965d-742b-466e-9baa-6983691506f6.png)

Is there a way to force it one way or another, so it does not wait for user's input?

**Comment by JanDeDobbeleer** [comment]
Good question. I'll provide a fix for that!

**Reply by JanDeDobbeleer** [reply]
Live!

**Reply by MikeyBronowski** [reply]
Noice, I will test it out soon :) Thanks!
